zanata-war/src/main/java/org/zanata/action/MergeTransAction.java

```diff
@@ -184,6 +186,9 @@ public void startMergeTranslations() {
         mergeTranslationsManager.start(sourceProjectSlug,
             sourceVersionSlug, targetProjectSlug, targetVersionSlug,
             !keepExistingTranslation);
+        FacesContext context = FacesContext.getCurrentInstance();
+        NavigationHandler navigationHandler = context.getApplication().getNavigationHandler();
+        navigationHandler.handleNavigation(context, null, "merge-translation");
```

Comment:
Ahh... so it won't apply the faces-config navigation rule from an ajax request to an action?